.eslintrc.js

```diff
@@ -0,0 +1,76 @@
+module.exports = {
+    extends: ['airbnb', 'prettier', 'prettier/flowtype', 'prettier/react'],
```

Comment:
> If prettier is getting real popular - which it lookes like - isn't there a popular eslint-config to match it?

Let's keep our eyes open. Maybe there will be new popular eslint config adjusted to work together with prettier, maybe airbnb will adjust theirs.

The airbnb ruleset focuses on stuff that help developers avoid common pitfalls. so it's complementary to prettier. they also have stuff for formatting obviously, but it certainly would make sense for the airbnb guys to simply remove all formatting rules overlapping with prettier from their ruleset.

For now, we have to do this tiny bit of duck-taping, but I also do not see a real pain this might cause. It's still just a (concatenated) list of rules–last rule wins.